cpp/arcticdb/entity/key.hpp

```diff
@@ -256,12 +256,6 @@ inline KeyType get_key_type_for_index_stream(const StreamId &) {
     return KeyType::TABLE_INDEX;
 }
 
-template <typename Function>
-auto foreach_key_type(Function&& func) {
-    for(int k = 0; k < int(KeyType::UNDEFINED); ++k) {
```

Comment:
It looks like the keytype = 0 is not used and also there is already a function to convert int to KeyType

Reply:
Yes, KeyTypes 1 and 2 were part of the original and now entirely superseded purpose or ArcticDB (as node storage for a specific graph strategy) and were never written anywhere.